todo_app/templates/index.html

```diff
@@ -11,8 +11,20 @@ <h1 class="display-4">To-Do App</h1>
     <div class="col-md-12">
       <h2>Items</h2>
       <ul class="list-group mb-4">
-        <!-- List the todo items here -->
+        {% for item in items %}
+        <li>{{ item.title }}</li>
+        {% endfor %}
       </ul>
     </div>
   </div>
+  <div class="row">
+    <div class="col-md-12">
+      <h2>Add an Item!</h2>
+      <form method="post" action="add">
+        <label for="title">Title:</label>
+        <input id="title" type="text" name="title"/>
```

Comment:
Really minor, but adding a `required` attribute here just to stop me accidentally submitting an empty to-do would be nice